README.md

```diff
@@ -10,6 +10,15 @@
 ember install ember-router-scroll
 ```
 
+### Options
+You can specify the id of an element for which the scroll position is saved and set. Default is `window` for using the scroll position of the whole viewport. You can pass an options object in your applications `config/environment.js` file.
```

Comment:
I have in the last commit: https://github.com/dollarshaveclub/ember-router-scroll/pull/67/commits/370bb90b28b628dbe0ba25ab6c549a480836bd73#diff-04c6e90faac2675aa89e2176d2eec7d8